sdks/java/core/pom.xml

```diff
@@ -429,6 +429,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
If you remove the above jsr305 dependency, you can probably get rid of the above `ignoredUnusedDeclaredDependency` config. I have this fix in a branch that isn't yet ready to merge-- perhaps I should get that in.